bin/public/installHaproxy.py

```diff
@@ -83,6 +81,15 @@ def install_haproxy(args):
     version_obj.mark_executed()
 
 
+def setup_global_vars():
+    """Initialize global variables from config files"""
+    global CERT_SERVER, CERT_SERVER_PATH, CERT_COPY_TO_PATH, SYCO_PLUGIN_PATH
```

Comment:
Since these are variables rather than constants (https://www.python.org/dev/peps/pep-0008/#constants) (their state is set at runtime) - they should not be uppercase but rather lowercase like other variables